data/mods/Aftershock/player/mutations/mastodon_mutation.json

```diff
@@ -0,0 +1,553 @@
+    "cost": 0
+  },
+  {
+    "type": "mutation",
+    "id": "FACEBUMPS",
+    "name": "Face Bumps",
+    "points": 0,
+    "visibility": 3,
+    "ugliness": 3,
+    "description": "You have a pair of bumps on your face above your mouth and beside your nose.",
+    "changes_to": [ "TUSKS" ]
+  },
+  {
+    "type": "mutation",
+    "id": "TUSKS",
+    "name": "Tusks",
+    "points": 1,
+    "visibility": 3,
+    "ugliness": 1,
+    "description": "You have a pair of small tusks on your face.  They allow you to make a weak piercing goring attack.",
+    "types": [ "HORNS" ],
+    "prereqs": [ "FACEBUMPS" ],
+    "changes_to": [ "TUSKS_POINTED" ],
+    "category": [ "MASTODON" ],
+    "attacks": {
+      "attack_text_u": "You gore %s with your tusks",
+      "attack_text_npc": "%1$s gores %2$s with their tusks",
+      "chance": 20,
+      "base_damage": [ { "damage_type": "stab", "amount": 3 }, { "damage_type": "bash", "amount": 3 } ]
+    }
+  },
+  {
+    "type": "mutation",
+    "id": "TUSKS_POINTED",
+    "name": "Pointed Tusks",
+    "points": 1,
+    "visibility": 8,
+    "ugliness": 2,
+    "mixed_effect": true,
+    "description": "You have a pair of long, pointed tusks, like someone weaponized an elephant.  They allow you to make a strong piercing headbutt attack, but prevent wearing mouthgear that is not made of fabric.",
+    "types": [ "HORNS" ],
+    "prereqs": [ "HORNS" ],
+    "restricts_gear": [ "MOUTH" ],
+    "allow_soft_gear": true,
+    "category": [ "MASTODON" ],
+    "threshreq": [ "THRESH_MASTODON" ],
+    "attacks": {
+      "attack_text_u": "You stab %s with your pointed tusks",
+      "attack_text_npc": "%1$s stabs %2$s with their pointed tusks",
+      "chance": 22,
+      "base_damage": { "damage_type": "stab", "amount": 24 }
+    }
+  },
+  {
+    "type": "mutation",
+    "id": "MUT_TANK",
+    "name": "Immovable",
+    "points": 2,
+    "description": "You can survive injuries that would incapacitate humans: you get a 20% bonus to all hit points.  Stacks with Tough, etc.",
+    "social_modifiers": { "intimidate": 2 },
+    "prereqs": [ "LARGE_OK", "HUGE_OK", "STR_UP_3", "STR_UP_4", "MASOCHIST_MED" ],
+    "threshreq": [ "THRESH_MASTODON" ],
+    "cancels": [ "FLIMSY", "FLIMSY2", "FLIMSY3", "GLASSJAW" ],
+    "changes_to": [ "MUT_TANK2" ],
+    "category": [ "MASTODON" ],
+    "hp_modifier_secondary": 0.2
+  },
+  {
+    "type": "mutation",
+    "id": "MUT_TANK2",
+    "name": "Unstoppable",
+    "points": 3,
+    "description": "Not much scares you.  You get a 30% bonus to all hit points.  Stacks with Tough, etc.",
+    "social_modifiers": { "intimidate": 3 },
+    "valid": false,
+    "prereqs": [ "MUT_TANK" ],
+    "threshreq": [ "THRESH_MASTODON" ],
+    "cancels": [ "FLIMSY", "FLIMSY2", "FLIMSY3", "GLASSJAW" ],
+    "changes_to": [ "MUT_TANK3" ],
+    "category": [ "MASTODON" ],
+    "hp_modifier_secondary": 0.3
+  },
+  {
+    "type": "mutation",
+    "id": "MUT_TANK3",
+    "name": "Superhero",
```

Comment:
So the whole tank line of mutations I'm using rewritten tough line mutations to avoid messing with other mods.  So Superhero isn't going to be specific to Mastodon line but rather be part of the alternative mutation tree we have to create until the `extend` value can be used in mutation categories.  I'm leaning towards renaming it "Impervious" if I haven't already used that in the tough replacement line.